README.md

```diff
@@ -24,6 +24,8 @@ Please see the link below for more detail to install Golang:
 
```

Comment:
The golang download link should be indented.

Reply:
The Getting started is missing the code snippet to list tables like Python and Node?

Reply:
The See Also section is missing items vs. the Python version?

Reply:
The editor does not seem to like to indent the link.  If I indent too much, it would stop becoming a link.
The code snippets are similar to the ones in the quick start guide, and Go has a link to that in #2 of See Also.
I added Go driver cookbook in #3.
There is no equivalent Go driver tutorial unlike with Python (https://docs.aws.amazon.com/qldb/latest/developerguide/getting-started.python.tutorial.html)
Also I can't find Go DMV sample Github repo (https://github.com/search?q=org%3Aaws-samples+dmv)

Reply:
See how Python has done it with using a bullet.

Reply:
Done

Reply:
@Dominitio The snippet for listing tables is still missing?